apps/src/templates/StylizedTabView.jsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import color from '@cdo/apps/util/color';
+
+/**
+ * StylizedTabView
+ */
+export default class StylizedTabView extends React.Component {
+  static propTypes = {
+    tabs: PropTypes.arrayOf(
+      PropTypes.shape({
+        key: PropTypes.string,
+        name: PropTypes.string,
+        renderFn: PropTypes.func
+      })
+    )
+  };
+
+  state = {
+    activeTab: this.props.tabs[0]
+  };
+
+  selectTab(tabInfo) {
+    this.setState({activeTab: tabInfo});
+  }
+
+  render() {
+    return (
+      <div>
+        <div style={styles.tabContainer}>
+          {this.props.tabs.map(tabInfo => (
+            <div
+              style={Object.assign(
+                {},
+                styles.tab,
+                this.state.activeTab === tabInfo
+                  ? styles.activeTab
+                  : styles.inactiveTab
+              )}
```

Comment:
I was trying to pass an array of styles and couldn't figure out why it wasn't working since I know we use it other places. Turns out that's also a Radium thing so it's fixed now.